test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -232,7 +233,7 @@ func (tc *CustomMetricTestCase) Run() {
 	projectID := framework.TestContext.CloudConfig.ProjectID
 
 	ctx := context.Background()
-	client, err := google.DefaultClient(ctx, gcm.CloudPlatformScope)
+	client, _ := google.DefaultClient(ctx, gcm.CloudPlatformScope)
```

Comment:
Why don't we add an error handling like line 251-253 instead of changing err to _?

Reply:
sure. Will do